MetaMask/metamask-extension
# Only check if a user has enough token balance before calling STX

## Explanation
Right now we are checking 2 things before calling the `getFees` function for STX on the View Quote page:
- Enough token balance for a swap
- Enough ETH balance for gas fees

With this PR we will only check if a user has enough token balance and then call STX, which will check if a user has enough ETH for gas fees. Also, if the `getFees` call returns an error now that a user doesn't have enough funds, we are falling back to regular swaps. With this PR if there is the "not enough funds" error from STX, we will show an info notification to a user, but keep STX active.

## Manual Testing Steps
- Try to make a swap with ETH balance you don't have
- On the Build Quote page enter almost all ETH balance you have, so you would have enough balance, but not enough ETH  for gas fees. E.g. if you have 0.012 ETH, try 0.011 ETH. You should see a notification about how much more ETH you need
